src/LocaleString.res

```diff
@@ -139,6 +140,7 @@ let defaultLocale = {
   completeNameEmptyText: str => `Please provide your complete ${str}`,
   billingDetailsText: "Billing Details",
   socialSecurityNumberLabel: "Social Security Number",
+  messageForWalletsDetailsSave: "Wallets details will be saved upon selection",
 }
```

Comment:
saveWalletDetails is fine for this 

keep the labels as precise as possible

Reply:
Sure